src/EditorFeatures/CSharpTest/SplitStringLiteral/SplitStringLiteralCommandHandlerTests.cs

```diff
@@ -37,18 +37,25 @@ public class SplitStringLiteralCommandHandlerTests
             Action callback,
             bool verifyUndo = true,
             IndentStyle indentStyle = IndentStyle.Smart,
-            bool useTabs = false)
+            bool useTabs = false,
+            string lineEnding = "\r\n")
         {
             using var workspace = TestWorkspace.CreateCSharp(inputMarkup);
             workspace.TryApplyChanges(workspace.CurrentSolution.WithOptions(workspace.Options
                 .WithChangedOption(SmartIndent, LanguageNames.CSharp, indentStyle)
-                .WithChangedOption(UseTabs, LanguageNames.CSharp, useTabs)));
+                .WithChangedOption(UseTabs, LanguageNames.CSharp, useTabs)
+                .WithChangedOption(NewLine, LanguageNames.CSharp, lineEnding)));
 
             if (useTabs && expectedOutputMarkup != null)
             {
                 Assert.Contains("\t", expectedOutputMarkup);
             }
 
+            if (lineEnding == "\n" && expectedOutputMarkup != null)
+            {
+                Assert.DoesNotContain("\r\n", expectedOutputMarkup);
```

Comment:
can you assert this for the inputMarkup as well?  And also the inverse (i.e. if the lineEnding is `\r\n` that we don't have naked `\n`s in the markup (can be checked with regexes).

Reply:
Done